dev_tools/autogenerate-bloqs-notebooks-v2.py

```diff
@@ -69,17 +70,21 @@
     NotebookSpecV2(
         title='Modular Exponentiation',
         module=qualtran.bloqs.factoring.mod_exp,
-        path_stem='mod_exp',
         bloq_specs=[qualtran.bloqs.factoring.mod_exp._MODEXP_DOC],
         directory=f'{SOURCE_DIR}/bloqs/factoring',
     ),
     NotebookSpecV2(
         title='Modular Multiplication',
         module=qualtran.bloqs.factoring.mod_mul,
-        path_stem='mod_mul',
```

Comment:
?

Reply:
The default is to match the module name. I must have had a revision where I was moving things around or doing something non-standard; but it's back to standard.